.eslintcache

```diff
@@ -1 +1 @@
\ No newline at end of file
+[{"D:\\AGATA\\Dare IT\\github_search\\src\\index.js":"1","D:\\AGATA\\Dare IT\\github_search\\src\\App.js":"2","D:\\AGATA\\Dare IT\\github_search\\src\\reportWebVitals.js":"3","D:\\AGATA\\Dare IT\\github_search\\src\\Components\\SearchBar.js":"4","D:\\AGATA\\Dare IT\\github_search\\src\\Components\\Results.js":"5"},{"size":500,"mtime":499162500000,"results":"6","hashOfConfig":"7"},{"size":234,"mtime":1611599690511,"results":"8","hashOfConfig":"7"},{"size":362,"mtime":499162500000,"results":"9","hashOfConfig":"7"},{"size":1370,"mtime":1611685710028,"results":"10","hashOfConfig":"7"},{"size":381,"mtime":1611685670166,"results":"11","hashOfConfig":"7"},{"filePath":"12","messages":"13","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"14"},"12j4t2v",{"filePath":"15","messages":"16","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"14"},{"filePath":"17","messages":"18","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"14"},{"filePath":"19","messages":"20","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"21","messages":"22","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"D:\\AGATA\\Dare IT\\github_search\\src\\index.js",[],["23","24"],"D:\\AGATA\\Dare IT\\github_search\\src\\App.js",[],"D:\\AGATA\\Dare IT\\github_search\\src\\reportWebVitals.js",[],"D:\\AGATA\\Dare IT\\github_search\\src\\Components\\SearchBar.js",[],"D:\\AGATA\\Dare IT\\github_search\\src\\Components\\Results.js",[],{"ruleId":"25","replacedBy":"26"},{"ruleId":"27","replacedBy":"28"},"no-native-reassign",["29"],"no-negated-in-lhs",["30"],"no-global-assign","no-unsafe-negation"]
```

Comment:
Let's keep it in .gitignore
https://www.pluralsight.com/guides/how-to-use-gitignore-file